src/PhyloTree.tsx

```diff
@@ -0,0 +1,183 @@
+            data: [
+                {
+                    name: 'tree',
+                    url: 'https://s3.amazonaws.com/gosling-lang.org/data/GeneSpy/gene_spy_tree.json',
+                    transform: [
+                        {
+                            type: 'stratify',
+                            key: 'id',
+                            parentKey: 'parent'
+                        },
+                        {
+                            type: 'tree',
+                            method: 'cluster',
+                            field: {field: 'distance'},
+                            sort: {field: 'value', order: 'ascending'},
+                            size: [{signal: 'height'}, {signal: 'width'}],
+                            separation: false,
+                            as: ['y', 'x', 'depth', 'children']
+                        },
+                        {
+                            type: 'formula',
+                            expr: 'width  * (datum.distance/' + maxDist + ')',
+                            as: 'x'
+                        }
+                    ]
+                },
+                {
+                    name: 'leaves',
+                    source: 'tree',
+                    transform: [
+                        {
+                            type: 'filter',
+                            expr: '!datum.children'
+                        }
+                    ]
+                },
+                {
+                    name: 'links',
+                    source: 'tree',
+                    transform: [
+                        {type: 'treelinks'},
+                        {
+                            type: 'linkpath',
+                            orient: 'horizontal',
+                            shape: 'orthogonal'
+                        }
+                    ]
+                }
+            ],
+            marks: [
+                {
+                    type: 'path',
+                    from: {data: 'links'},
+                    encode: {
+                        update: {
+                            path: {field: 'path'},
+                            stroke: {value: '#000'}
+                        }
+                    }
+                },
+                {
+                    type: 'rule',
+                    from: {data: 'leaves'},
+                    encode: {
+                        update: {
+                            x: {field: 'x'},
+                            y: {field: 'y'},
+                            x2: {value: localWidth},
+                            y2: {field: 'y'},
+                            stroke: {value: '#eee'},
+                        }
+                    }
+                },
+                {
+                    type: 'text',
+                    from: {data: 'leaves'},
+                    encode: {
+                        enter: {
+                            text: {field: 'name'},
+                            fontSize: {value: 9},
```

Comment:
If you want to add styling options for the tree, I think the font size would be quite useful!